cmd/glasskube/cmd/bootstrap.go

```diff
@@ -97,12 +111,55 @@ func (o bootstrapOptions) asBootstrapOptions() bootstrap.BootstrapOptions {
 	}
 }
 
+func (o bootstrapOptions) printBootsrap(manifests []unstructured.Unstructured, output OutputFormat) error {
```

Comment:
the output format is used twice here but taken from different places.

I think the `bootstrapOptions` receiver should be removed here and then the function should check `if output != ""` instead.